search/query/date_range.go

```diff
@@ -30,10 +30,10 @@ import (
 	index "github.com/blevesearch/bleve_index_api"
 )
 
-// QueryDateTimeParser controls the default query date time parser
+// QueryDateTimeParser controls the default query date time parser in User Path.
```

Comment:
Not a fan of the "User path" wording - I understand what you are intending it for, but I think the name of the attribute is self-explanatory enough. let's drop the "User Path" verbiage.

Reply:
Also one rule of thumb - do not add anything "couchbase" specific here. Keep things generic enough.